extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/SceneHandler.java

```diff
@@ -112,47 +112,43 @@ public void handleRemoval() {
     }
 
     @Override
-    public void bridgeHandlerInitialized(ThingHandler thingHandler, Bridge bridge) {
-        if (thingHandler instanceof BridgeHandler) {
-            this.bridgeHandler = (BridgeHandler) thingHandler;
-
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
-                    "waiting for listener registration");
-            logger.debug("Set status on {}", getThing().getStatus());
-
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        if (bridgeStatusInfo.getStatus().equals(ThingStatus.ONLINE)) {
+            if (getBridgeHandler() != null) {
+                String sceneID = getSceneID(getConfig(), bridgeHandler);
+                switch (sceneID) {
+                    case SCENE_WRONG:
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                                "Configured scene '" + getConfig().get(DigitalSTROMBindingConstants.SCENE_ID)
+                                        + "' does not exist or cannot be used, please check the configuration.");
+                        break;
+                    case ZONE_WRONG:
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                                "Configured zone '" + getConfig().get(DigitalSTROMBindingConstants.SCENE_ZONE_ID)
+                                        + "' does not exist, please check the configuration.");
+                        break;
+                    case GROUP_WRONG:
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                                "Configured group '" + getConfig().get(DigitalSTROMBindingConstants.SCENE_GROUP_ID)
+                                        + "' does not exist, please check the configuration.");
+                        break;
+                    case NO_STRUC_MAN:
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
```

Comment:
At least the documentation (https://github.com/eclipse/smarthome/blob/master/docs/documentation/concepts/things.md) states that CONFIGURATION_PENDING is valid for ONLINE.